pkg/export/otel/metrics.go

```diff
@@ -180,7 +181,7 @@ func getMetricEndpointOptions(cfg *MetricsConfig) ([]otlpmetrichttp.Option, erro
 	if murl.Scheme == "http" || murl.Scheme == "unix" {
 		opts = append(opts, otlpmetrichttp.WithInsecure())
 	}
-	if len(murl.Path) > 0 && murl.Path != "/" {
+	if len(murl.Path) > 0 && murl.Path != "/" && !strings.HasSuffix(murl.Path, "/v1/metrics") {
```

Comment:
Hmmm I'm thinking in the case of someone willing to pass:

`OTEL_EXPORTER_OTLP_METRICS_ENDPOINT=https://foo:8080/my/metrics`, then the code will set the endpoint as `https://foo:8080/my/metrics/v1/metrics`. But it should remain untouched in that case, right?

I think then the code should be something like:

```go
if cfg.MetricsEndpoint == "" && murl.Path != "/" {
    opts = append(opts, otlpmetrichttp.WithURLPath(murl.Path+"/v1/metrics"))
}
```